### PR TITLE
Docs: Update Okta SSO Guide

### PR DESCRIPTION
**Pull request by alexfornuto**
Update to resolution for #21987.

Updates include:

- Use `tctl sso configure` to generate connector resource
- Updates Okta app config and screenshot to match defaults provided by ^ (includes port number)
- Updates partial used throughout saml guides to:
  - Default to the dynamic configuration tab for all versions
  - include a note on how to log in locally before saml connector is configured.
- Update `allow_idp_initiated` partial with full path to key, since configs generated by `tctl sso configure` don't include the key by default.

### CODE REVIEW

File: docs/pages/access-controls/sso/okta.mdx [hunk 1]
```diff
@@ -134,22 +144,97 @@ You can also right click on the "View IdP metadata" link and select
...
+version: v2
+```
 
 The `attributes_to_roles` field in the connector resource maps key/value-like attributes of
 the assertion from Okta into a list of Teleport roles to apply to the session.
 
 (!docs/pages/includes/sso/idp-initiated.mdx!)
 
-Create the connector using `tctl` tool:
+You can optionally test the connector before applying it to your cluster:
```
Comment (Tener) [inline docs/pages/access-controls/sso/okta.mdx:193]: This is very likely to catch errors, so testing is advised.

File: docs/pages/access-controls/sso/okta.mdx [hunk 2]
```diff
@@ -134,22 +144,97 @@ You can also right click on the "View IdP metadata" link and select
...
+Define an Okta SAML connector using `tctl`. Update this example command with
+the path to your metadata file, and edit the `--attributes-to-roles` values for
+custom group assignment to roles. See [tctl sso configure
+saml](../../reference/cli.mdx#tctl-sso-configure-saml) for a full reference of
+flags for this command:
 
-```yaml
-(!examples/resources/saml-connector.yaml!)
+```code
+$ tctl sso configure saml --name=okta \
```
Comment (Tener) [inline docs/pages/access-controls/sso/okta.mdx:154]: We do have `--preset` flag and `okta` is one of the allowed values. If you choose a preset, `--name` is no longer required: https://github.com/gravitational/teleport.e/blob/f7f187a26028f7144404abae5e5ef44d395bd195/tool/tctl/sso/configure/saml.go#L166-L168

In addition to setting a name, the preset can modify the settings in a particular way. Right now the only non-cosmetic change happens for `adfs`. We still set `description` and `display` values. 

See also: https://github.com/gravitational/teleport.e/blob/f7f187a26028f7144404abae5e5ef44d395bd195/tool/tctl/sso/configure/saml.go#L51-L59

**Review by Tener — APPROVED** [review]

File: docs/pages/access-controls/sso/okta.mdx (hunk 2 above)
Comment (alexfornuto) [reply docs/pages/access-controls/sso/okta.mdx]: Resolved in 7565debd0b8fafc9c5ab865b7e222a63f013cca9

File: docs/pages/access-controls/sso/okta.mdx [hunk 3]
```diff
@@ -23,7 +23,16 @@ Teleport administrators to define policies like:
 
 - A Teleport role with access to edit and maintain `saml` resources. This is
   available in the default `editor` role.
-- (!docs/pages/includes/tctl.mdx!)
+
+<Tabs>
+<TabItem label="Teleport Enterprise (Self-Hosted)" scope="enterprise">
+- (!docs/pages/includes/enterprise/tctl-tsh-prerequisite.mdx!)
```
Comment (ptgott) [inline docs/pages/access-controls/sso/okta.mdx:29]: This partial isn't being included for some reason

File: docs/pages/access-controls/sso/okta.mdx [hunk 4]
```diff
@@ -23,7 +23,16 @@ Teleport administrators to define policies like:
...
+</TabItem>
+<TabItem label="Teleport Cloud" scope="cloud">
+- (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
```
Comment (ptgott) [inline docs/pages/access-controls/sso/okta.mdx:32]: Same partial is not being included here

**Review by ptgott — APPROVED** [review]
A partial isn't being included, but since the linter is blocking on this anyway, I thought I'd approve

File: docs/pages/access-controls/sso/okta.mdx (hunk 3 above)
Comment (alexfornuto) [reply docs/pages/access-controls/sso/okta.mdx:29]: Good catch! Resolved with updates from `master`.

**Pull request merged**